sdk/scheduler/src/test/java/com/mesosphere/sdk/specification/TestPodFactory.java

```diff
@@ -116,6 +113,30 @@ public static ResourceSet getResourceSet(String id, double cpu, double mem, doub
                 .build();
     }
 
+    public static PodSpec getMultiTaskPodSpec(
```

Comment:
Maybe just put this in `DefaultPlanCoordinatorTest.java` given that's the only place that it's being called